importer/RsSr.py

```diff
@@ -0,0 +1,129 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class RsSr(Monument):
+
+    def set_address(self):
+        """
+        Set address / directions.
+
+        Address, with attached city, if there's digits in it.
+        Otherwise directions.
+        """
+        if self.has_non_empty_attribute("address"):
+            if self.has_non_empty_attribute("city"):
+                if utils.contains_digit(self.address):
+                    city_no_brackets = utils.get_rid_of_brackets(self.city)
+                    address = "{}, {}".format(self.address, city_no_brackets)
+                    self.add_statement("located_street", address)
+                else:
+                    dirs = utils.package_monolingual("sr", self.address)
+                    self.add_statement("directions", dirs)
+
+    def set_location(self):
+        """
+        Set location property.
+
+        'City' is never wikilinked, so try to match
+        against list of known settlements.
+        """
+        if self.has_non_empty_attribute("city"):
+            loc_q = None
+            loc_raw = self.city
+            loc_dic = self.data_files["settlements"]
+
+            loc_match = utils.get_item_from_dict_by_key(dict_name=loc_dic,
+                                                        search_in="itemLabel",
+                                                        search_term=loc_raw)
+            if len(loc_match) == 1:
+                loc_q = loc_match[0]
+
+            if loc_q:
+                self.add_statement("location", loc_q)
+            else:
+                self.add_to_report("city", self.city, "location")
+
+    def set_adm_location(self):
+        """
+        Set administrative location.
+
+        If possible, match district / municipality.
+        Not wikilinked, and contain a mix of various
+        types, so match against list of all administrative
+        units.
+        If that doesn't work, resort to okrug via iso.
+        """
+        adm_q = None
+        adm_dic = self.data_files["admin"]
+        okr_dic = self.data_files["okruzi"]
+
+        municip_try = utils.get_item_from_dict_by_key(
+            dict_name=adm_dic,
+            search_in="itemLabel",
+            search_term=self.district)
+        if len(municip_try) == 1:
+            adm_q = municip_try[0]
```

Comment:
`else: self.add_to_report("district", self.district, "located_adm")` (if seld.district is never empty, else might want to use `elif self.dirstrict`

Reply:
self.district is never empty. done.